man/sd_bus_message_open_container.xml

```diff
@@ -100,8 +100,13 @@
   <refsect1>
     <title>Return Value</title>
 
-    <para>On success, these functions return a non-negative integer. On failure, they return a negative
-    errno-style error code.</para>
+    <para>On success, these functions return a non-negative integer.
+    <function>sd_bus_message_open_container()</function> and <function>sd_bus_message_close_container()</functiony>
```

Comment:
Please preview the man page with `build/man/man sd_bus_message_open_container`. XML is invalid here: "functiony".